nba/data_wrangling.py

```diff
@@ -0,0 +1,161 @@
+import pandas as pd
+import numpy as np
+from pathlib import Path
+import sqlite3 as sql
+
+# Set the data path (adjust if necessary)
+DATA_PATH = Path.cwd() / "data" / "nba.sqlite"
+
+
+# Data loading function for insights 1 & 2

```

Comment:
Hey,
I think load_data, load_data_draft, load_data_team should be the same function, we can just use different table name as parameter?